ui/DesignSystem/Component/SourceBrowser/History.svelte

```diff
@@ -1,11 +1,14 @@
 <script lang="typescript">
   import { createEventDispatcher } from "svelte";
 
-  import type { CommitHeader, CommitsHistory } from "../../../src/source";
+  import type {
+    CommitHeader,
+    GrouppedCommitsHistory,
```

Comment:
```suggestion
    GroupedCommitsHistory,
```

and the same/similar typos in both `source.ts` files